### DISCUSSION TITLE
Gathering more information regarding application.

### DISCUSSION THREAD

**Discussion by MR-ENVYR** [open-edge-platform/anomalib#949] [General]
Hello.
I am Raghavendra and I am a research student working with deep anomaly detection models. I came across Anomalib recently and found that it would be a great library to help with my research and felt that I should contribute to it. I noticed that it is recommended to get in touch with the mentors before applying so I wanted to discuss a few possibilities and ideas before finishing the prerequisite task and putting the application. Would this be a good place to discuss or should I shift to mail. Please let me know about the same.

Thanks in advance!  :)

**Comment by ashwinvaidya17** [comment]
Great to know that you are interested in contributing. This discussion page is the best way to discuss your ideas. If however you feel like discussing something more private feel free to reach out to one of the mentors by email.
